src/v2/view-builder/views/on-prem/ChallengeAuthenticatorOnPremView.js

```diff
@@ -0,0 +1,40 @@
+import { loc } from 'okta';
+import BaseForm from '../../internals/BaseForm';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import AuthenticatorEnrollFooter from '../../components/AuthenticatorEnrollFooter';
+
+const ON_PREM_TOKEN_CHANGE_KEY = 'idx.error.on-prem.token.change';
+
+const Body = BaseForm.extend({
+
+  className: 'on-prem-authenticator-verify',
+
+  modelEvents: {
+    'error': '_checkTokenChange'
+  },
+
+  title () {
+    const vendorName =
+      this.options.appState.get('currentAuthenticatorEnrollment').displayName ||
+      loc('oie.on_prem.authenticator.default.vendorName', 'login');
+    return loc('oie.on_prem.verify.title', 'login', [vendorName]);
+  },
+
+  _checkTokenChange (model, convertedErrors) {
+    const errorSummaryKeys = convertedErrors?.responseJSON?.errorSummaryKeys;
+    if (errorSummaryKeys && errorSummaryKeys.includes(ON_PREM_TOKEN_CHANGE_KEY)) {
+      // this means we are in change pin, so we should clear out answer input
+      this.model.set('credentials.passcode', '');
```

Comment:
Why are we having to clear and rerender the whole form. We don't do this for other anyother totp  based verify or enrolll

Reply:
Can we double check with UX/PM and remove this if they are okay.

Reply:
Yea, i checked. This is there in v1 too. So they want parity

Reply:
In v1, it's checking error status code (409), are you able to do same ?